virus/sedimentum_occultus.md

```diff
@@ -0,0 +1,111 @@
+
+The two BLAST hits for ORF2 with the highest percent identity were structural proteins. The first one was a putative structural protein from the *Picornavirales* order, which the *Picornaviridae* family belongs to. The associated publication for this protein sought to determine virus--host relationships in marine systems. The authors noticed that they collected the most transcripts of unclassified *Picornavirales* viruses during brown tide blooms of the algae *Aureococcus anophagefferens* in Quantuck Bay, New York, which was also associated with a rise in diatoms, dinoplagellates, and other protists. This suggested that one or several of these organisms may be the host of some unknown viruses from the *Picornavirales* order, but the authors could not conclude which [Moniruzzaman et al. 2017]. In terms of *S. occultus*, it is possible that its host may be a similar marine organism, but this is inconclusive because the viruses found in the study were not from the same location and may not have been from the same family. The second BLAST hit was a study on the scallop *Argopecten irradians* in the Gulf coast of Florida that revealed three novel viruses in the family *Picornaviridae* within the scallop pathobiome [Scro et al. 2023]. These two studies taken together suggest that it may be possible that the host of *S. occultus* may be a marine animal or protist, as viruses with similar caspid proteins have been found as the putative hosts of these organisms. However, it is still inconclusive as to what the host of *S. occultus* is.
+
+### Virus Genome
+
+Still Image:
+
+![](img/sedimentum_occultus/figure_1.png)
+
+[Interactive Version](https://www.ncbi.nlm.nih.gov/projects/sviewer/?id=lcl|1&tracks=[key:sequence_track,name:Sequence,display_name:Sequence,id:STD649220238,category:Sequence,annots:Sequence,ShowLabel:false,ColorGaps:false,shown:true,order:0][key:gene_model_track,name:U2EZpwA2z898C,display_name:(U)%20ORFfinder_12.4.72257490,id:U2EZpwA2z898C,data_key:dcrjGxXIwunu5vTuxffK4Jmdm5yTnr-Ut4y5piyrKgnqPwqyJwYWXVoPZHYxamhyelotW25FflRkTnRRQnx7fltOfQ,dbname:NetCache,category:ORFfinder,annots:ORFfinder_12.4.72257490_UUD1701692577DUU_gene_model,Options:MergeAll,CDSProductFeats:false,NtRuler:true,AaRuler:true,HighlightMode:2,ShowLabel:true,highlights_color:#0000ff,shown:true,order:1]&key=_kFokJ5DSWJlbX9lTnxBaxIWEBcYFTQfPAcyLacgoYJitBa9sNaB6w-kRN0RwUjZWvEN8E7uXv9E5VT6Ytdb1XvlXQ&mk=3479:3502|C|red,3797:3832|A|0000FF,3584:3625|B|00FF00,2969:8341|ORF1|black|9,247:2679|ORF2|333333|1,3050:4282|RdRp|FF00FF&noguess_assm=1&spacing=1&v=1:8971&c=null&gflip=1&select=null&slim=0&toolbar=-d) (may not work)
```

Comment:
The interactive versions are not likely to be stable over time (unless you know they are). Please remove them as they will be dead links.